libraries/AP_HAL_ChibiOS/hwdef/CUAV-Nora/hwdef.dat

```diff
@@ -233,6 +234,9 @@ SPIDEV rm3100         SPI2 DEVID2  RM3100_CS       MODE3  2*MHZ  8*MHZ
 SPIDEV icm20649       SPI6 DEVID2  ICM20649_CS       MODE3  2*MHZ  8*MHZ
 SPIDEV icm20689_board SPI6 DEVID2  ICM20689_BOARD_CS MODE3  2*MHZ  8*MHZ
 
+#Mount bmi088_a or icm42688 on SPI4
+SPIDEV bmi088_a       SPI4 DEVID2  BMI088_A_CS     MODE3  10*MHZ 10*MHZ
```

Comment:
this gives a duplicate SPIDEV line for bmi088_a, we should only have each SPIDEV once